### PR TITLE
restart kubelet after node join

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: install_scripts/templates/kubernetes/node-join.sh [hunk 1]
```diff
@@ -99,6 +99,21 @@ promptForAddress() {
...
 }
 
+# Once the Rook agent has started and copied its plugins to the host, the
+# the kubelet may need to be restarted as a fix for a race condition in
+# K8s 1.11 and Rook < 0.9. https://github.com/rook/rook/issues/2064
+ensureRookPluginsRegistered() {
+    logStep "Await Node Ready"
+    # If Rook is enabled it should start around the same time Weave starts.
+    # Cannot wait for Rook directly because it may be disabled.
+    while ! docker ps | grep weave-net > /dev/null ; do
```
Comment (emosbaugh) [inline install_scripts/templates/kubernetes/node-join.sh]: cant you `grep -q`?

File: install_scripts/templates/kubernetes/node-join.sh [hunk 2]
```diff
@@ -99,6 +99,21 @@ promptForAddress() {
...
+        sleep 2
+    done
+    sleep 10
+    sudo systemctl restart kubelet
```
Comment (emosbaugh) [inline install_scripts/templates/kubernetes/node-join.sh]: do you need sudo?

File: install_scripts/templates/kubernetes/node-join.sh (hunk 1 above)
Comment (areed) [reply install_scripts/templates/kubernetes/node-join.sh]: right

File: install_scripts/templates/kubernetes/node-join.sh (hunk 2 above)
Comment (areed) [reply install_scripts/templates/kubernetes/node-join.sh]: no

**Pull request merged**